usaspending_api/search/tests/integration/hierarchical_filters/es_search_test_helpers.py

```diff
@@ -0,0 +1,60 @@
+import json
+from django.conf import settings
+
+from usaspending_api.search.elasticsearch.filters.tas import TasCodes, TreasuryAccounts
+from usaspending_api.common.experimental_api_flags import EXPERIMENTAL_API_HEADER, ELASTICSEARCH_HEADER_VALUE
+
+
+def _setup_es(client, monkeypatch, elasticsearch_award_index):
```

Comment:
If you have time would be nice to replace this and its usage with the `setup_elasticsearch_test()` function defined under `usaspending_api/search/tests/data/utilities.py`. It will setup elasticsearch according to the type of index you pass it.

Reply:
Just implemented a simple refactor. When I circle back to write more tests for the transaction index, I'll imrpove this further.